src/components/IFrame.js

```diff
@@ -17,73 +35,155 @@ function getNewDotURL(url) {
     }
 
     if (pathname === 'inbox') {
-        return 'home';
+        return ROUTES.HOME_OLDDOT;
     }
 
     if (pathname === 'expenses') {
-        return `${params.viewMode === 'charts' ? 'insights' : 'expenses'}${paramString ? `/?param=${paramString}` : ''}`;
+        return `${params.viewMode === 'charts' ? ROUTES.INSIGHTS_OLDDOT : ROUTES.EXPENSES_OLDDOT}${paramString ? `/?param=${paramString}` : ''}`;
     }
 
     if (pathname === 'admin_policies') {
         const {section} = params;
-        return section === 'individual' ? 'individual_workspaces' : 'group_workspaces';
+        return section === 'individual' ? ROUTES.INDIVIDUALS_OLDDOT : ROUTES.GROUPS_OLDDOT;
     }
 
     if (pathname === 'policy') {
-        const workspaceID = params.policyID || '';
-        const section = urlObj.hash.slice(1) || 'overview';
+        const workspaceID = params.policyID ?? '';
+        const section = urlObj.hash.slice(1) ?? '';
+
+        let sectionName = section;
+        switch (section) {
+            case '':
+                sectionName = 'overview';
```

Comment:
Let's make the 'overview' string value a constant since we're using it in multiple places.